src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newObject = {};
+  const clearString = sourceString.trim();
+  const separation = clearString.split(';');
```

Comment:
[CODE KNOWLEDGE] - You can chain string and array methods to avoid unnecessary intermediate variables. For example, you can chain `trim`, `split`, and `forEach` directly without assigning them to variables like `clearString` and `separation`.